docs/source/index.rst

```diff
@@ -11,69 +11,18 @@ JupyterLab is the next-generation web-based user interface for Project Jupyter.
 
 .. image:: ./images/jupyterlab.png
    :align: center
-   :class: jp-screenshot
+   :class: jp-screenshot shadow
 
+Get started
+===========
 
+See the sections below (and the top-level links above) for more information about using, extending, and contributing to JupyterLab.
 
 .. toctree::
-   :maxdepth: 1
-   :caption: Getting Started
+   :maxdepth: 2
+   :caption: Table of Contents
```

Comment:
Sorry to change my mind. As the caption is not highlighted, I think it is confusing. So your first proposal without caption sounds great.

![image](https://user-images.githubusercontent.com/8435071/149085123-c31cfba3-24c7-4f6c-a55c-f13ee20e0b91.png)


```suggestion
```